Explorer/Assets/Scripts/Global/Dynamic/DynamicWorldContainer.cs

```diff
@@ -165,6 +167,11 @@ private static void BuildReloadSceneWidget(IDebugContainerBuilder debugBuilder,
 
             IDebugContainerBuilder debugBuilder = dynamicWorldDependencies.DebugContainerBuilder;
 
+            // TODO: it might be useful to cache the third-party nft providers info at start so we avoid undesired delays in the UI.
```

Comment:
Agreed! 
I would remove the TODO comment to avoid confusion, because there is nothing to do. If you want to keep it, I would paraphrase it to somehting like "If we start having lots fo thrid party wearable collection, it might be useful..."